foldamers/utilities/util.py

```diff
@@ -727,7 +730,7 @@ def get_random_positions(
                             # this choice not working now
                             trial_positions, placement = assign_position(
                                 stored_positions,
-                                monomer_type["bond_lengths"],
+                                get_bond_length[bond],
```

Comment:
Should this be get_bond_length(bond)?

Reply:
indeed, didn't notice it since that code path doesn't work right now.